src/MooLite/plugins/MooLite/MooLiteClientPlugin.ts

```diff
@@ -63,6 +71,19 @@ export class MooLiteClientPlugin extends MooLitePlugin {
             type: PluginConfigType.CheckBox,
             value: false,
         },
+        {
+            key: "mooliteclient/display-side",
+            name: "MooLite display side",
+            description: "Display MooLite on the left or the right",
+            type: PluginConfigType.Choice,
+            value: "right",
+            options: mooLiteDisplaySides.map((side) => {
+                return {
+                    text: toTitleCase(side),
+                    value: side,
+                };
+            }),
```

Comment:
I think this could improve dropdown configs if it was more generic e.g. `options: mapArrayToOptions(mooLiteDisplaySides)`, and save some of the verbosity around using enums alongside their names